lib/ddtrace/contrib/concurrent_ruby/patcher.rb

```diff
@@ -16,11 +16,20 @@ def target_version
         def patch
           require 'ddtrace/contrib/concurrent_ruby/future_patch'
           patch_future
+          require 'ddtrace/contrib/concurrent_ruby/promises_future_patch'
+          patch_promises_future
         end
 
         # Propagate tracing context in Concurrent::Future
         def patch_future
-          ::Concurrent::Future.send(:include, FuturePatch)
+          ::Concurrent::Future.send(:prepend, FuturePatch) if defined?(::Concurrent::Future)
+        end
+
+        # Propagate tracing context in Concurrent::Promises::Future
+        def patch_promises_future
+          if defined?(::Concurrent::Promises::Future)
+            ::Concurrent::Promises.singleton_class.send(:prepend, PromisesFuturePatch)
```

Comment:
We can avoid all `.send(:prepend, ... `) since we no longer need to support 2.0 (see #1509). I'm assuming this is one of the reasons why there's a merge conflict with this PR.

Reply:
Alright, seeing the conflict that's what I guessed. Thanks for the reference, updating!